configure.cmd

```diff
@@ -1,103 +0,0 @@
-@ECHO OFF
-SETLOCAL ENABLEDELAYEDEXPANSION
-
-REM Batch script reminder: variables between % % are evaluated as parse time, not at runtime.
-REM    Variables set dynamically (including being read from a file) should be evaluated
-REM    using ! ! instead of % %. However, this only allows one level of expansion. Do not
-REM    try to create a variable that derives from a derived variable. It will be empty.
-
-REM First Check that Deno isn't running since this causes weird and confusing errors
-tasklist /fi "ImageName eq deno.exe" /fo csv 2>NUL | find /I "deno.exe">NUL
-if "%ERRORLEVEL%"=="0" goto :denoRunning
```

Comment:
Unfortunately, it probably is worthwhile. Thanks for catching that. The issue is that on Windows, you can't overwrite a file that is currently open. This interacts strongly with the whole "reconfiguration" scheme, and I'm not sure how that ever worked. As I understand it, if the deno program called back into the shell/batch script as a subprocess, then deno would always be running, and the reconfiguration would never have worked on windows. I need to go back and test that to understand how it worked (or didn't).

Reply:
I can confirm that the reconfiguring worked well when used in RStudio IDE where I am mostly using Quarto. 
So I did not encountered the issue with Deno LSP before the check was introduced. I can't say. 

I think it is worth adding back, but it means indeed that the reconfiguring won't work when using quarto inside VSCODE

Reply:
Right, though I think the idea was that before this check, reconfiguring when working in VSCode on quarto didn't work anyway (since the Deno LSP would be running, making deno.exe `open`), but threw a confusing message without much help as to how to resolve. This at least tries to provide some clue (I added this as I only worked in windows occasionally and I would always get bitten by this and spend 15 minutes trying to debug why the error was happening before I realized I should just close vscode).